plugins/generic/metadataExport/metadataExportFormats/bibtex/BibtexMetadataExportPlugin.inc.php

```diff
@@ -0,0 +1,98 @@
+<?php
```

Comment:
Rather than duplicating all the existing citation format plugins in a new category, is it not possible to just re-use the existing citation format plugins?